bin/improver-blend-adjacent-points

```diff
@@ -83,7 +83,7 @@ def main():
 
     if args.coordinate == 'time':
         parameter_unit = Unit(args.parameter_unit, args.calendar)
```

Comment:
default for args.calendar doesn't appear to be being set.

Reply:
Responded.